acceptance/tests/options/config_file/no_custom_facts_and_custom_dir.rb

```diff
@@ -0,0 +1,43 @@
+# This test verifies that setting both custom-dir and no-custom-facts results in an error
+test_name "99994: config option no-custom-facts conflicts with custom-dir" do
```

Comment:
Should the testcase number be, _C99994_? You've used the leading _C_ pretty consistently elsewhere.